src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -51,7 +51,9 @@ getCurrentHub()
   .addIntegration(replay);
 
 // sometime later
+await replay.flush(); // wait for any pending recording data to be sent
```

Comment:
Wonder if this should be part of `stop()`

Reply:
IMHO it's fine to have this be separate.